external/CMakeLists.txt

```diff
@@ -61,6 +61,8 @@ list(APPEND EXTERNAL_INCLUDE_DIRS ${LIB_STATIC_APRUTIL1_INC_DIR})
 
 include(YamlLib)
 set(LIB_STATIC_YAML_LOC ${LIB_STATIC_YAML_LOC} PARENT_SCOPE)
+set(LIB_STATIC_YAML_INC_DIR ${LIB_STATIC_YAML_INC_DIR} PARENT_SCOPE)
```

Comment:
`LIB_STATIC_YAML_INC_DIR` doesn't seem to be used anywhere outside of this module, so not clear what purpose this statement serves.